diff --git a/src/ngrok/server/http.go b/src/ngrok/server/http.go
index 6273658..4a027c2 100644
--- a/src/ngrok/server/http.go
+++ b/src/ngrok/server/http.go
@@ -22,7 +22,7 @@ Authorization required
 	NotFound = `HTTP/1.0 404 Not Found
 Content-Length: %d
 
-Tunnel %s not found
+Tunnel not found
 `
 
 	BadRequest = `HTTP/1.0 400 Bad Request
@@ -91,7 +91,7 @@ func httpHandler(c conn.Conn, proto string) {
 	tunnel := tunnelRegistry.Get(fmt.Sprintf("%s://%s", proto, host))
 	if tunnel == nil {
 		c.Info("No tunnel found for hostname %s", host)
-		c.Write([]byte(fmt.Sprintf(NotFound, len(host)+18, host)))
+		c.Write([]byte(NotFound))
 		return
 	}
 
